### PR TITLE
Merges whisper verb into saycode, extends to all living mobs

### LINKED ISSUE #25657: Aliens cannot succumb by whispering or deathgasping

### PR DESCRIPTION
**Pull request by JJRcop**
:cl:
tweak: Refactors whisper, now all living mobs can use it
add: Prefix any message with # to whisper, for example, "# Man, that guy is weird.", you can still use the whisper verb if you whish.
add: This is compatible with languages, for example "#,r That HoS is a problem, we should kill him"
del: Removes #a #b #c etc radio codes, as well as :w and .w, which used to be whisper.
/:cl:

Removes human whisper verb and integrates the functionality into mob/living saycode.
Removed all #a #b #c radio codes.
Removed whisper radio codes :W, .w etc.
The reason for this is you can now prefix messages with # to make them whisper instead, the whisper verb still exists, but now just calls ```say("#[message]")```.

This is fully compatible with language datums, clock cultists can whisper in ratvarian with ```"#,r Let's get the chaplain."```, or just use the whisper verb.

Closes #25938 
Addresses half of #25657, at least the whisper part. deathgasp is not intended to work, and the reason succumb doesn't work is another issue.

### CODE REVIEW

File: code/modules/mob/living/say.dm [hunk 1]
```diff
@@ -129,10 +131,26 @@ var/list/crit_allowed_modes = list(MODE_WHISPER,MODE_CHANGELING,MODE_ALIEN)
...
+		spans |= SPAN_ITALICS
+		log_whisper("[src.name]/[src.key] : [message]")
+		if(in_critical)
+			var/health_diff = round(-HEALTH_THRESHOLD_DEAD + health)
+			// If we cut our message short, abruptly end it with a-..
+			var/message_len = length(message)
+			message = copytext(message, 1, health_diff) + "[message_len > health_diff ? "-.." : "..."]"
+			message = Ellipsis(message, 10, 1)
+			message_mode = MODE_WHISPER_CRIT
+			succumb(1)
```
Comment (ChangelingRain) [inline code/modules/mob/living/say.dm]: You need to do this after sending the message, as otherwise the mob may not exist.

**Comment by PKPenguin321** [comment]
Will this affect whisperings interaction with voice analyzers? I'd rather if it didn't, kind of a balance change

**Comment by coiax** [comment]
I don't know if anyone used # as a radio key, but it should definitely be in the changelog.

**Comment by JJRcop** [comment]
I completely forgot about the changelog... oops

File: code/modules/mob/living/say.dm [hunk 2]
```diff
@@ -1,76 +1,79 @@
...
 		message = trim(copytext(sanitize(message), 1, MAX_MESSAGE_LEN))
 	if(!message || message == "")
 		return
 
 	var/message_mode = get_message_mode(message)
 	var/original_message = message
+	var/in_critical = InCritical()
 
-	if(message_mode == MODE_HEADSET || message_mode == MODE_ROBOT)
+	if(message_mode in one_character_prefix)
```
Comment (RemieRichards) [inline code/modules/mob/living/say.dm]: never use ```in```. make your lists assoc.

**Comment by ChangelingRain** [comment]
Please don't reduce the default message range.

**Comment by JJRcop** [comment]
I reduced it because it extends one tile further than the specified range, and people hear the text starred at that extra tile.

**Comment by lzimann** [comment]
That's kinda of a balance change though.

**Comment by ExcessiveUseOfCobblestone** [comment]
passing your balance changes under a refactor, FOR SHAME

**Comment by optimumtact** [comment]
LGTM

**Pull request merged**